jupyterlab_comments/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I recommend referencing the top-level tsconfig file like https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/jupyterlab_gcpscheduler/tsconfig.json.

We also have a tsconfig.prod.json in each package to exclude any test sources from the production build.